### ISSUE TITLE
iOS 14 - carplay issues

### ISSUE THREAD

**Issue by avxkim** [2gis/MapGL-iOS#5]
Whenever i start 2gis in a carplay mode, it shows a blank map, even after 30 minutes. Nothing helps. Have you tested your application on ios14/carplay?

**Comment by teanet** [comment]
We have issues on iOS 14 with WebGL rendering, working on it currently.

**Comment by avxkim** [comment]
Please, update this issue, when you're done, so i will migrate to ios 14.

**Comment by AlejandroRZ1** [comment]
Yes also the signal is not working and now it’s getting my phone to the point of proving internet

**Comment by teanet** [comment]
```
window.map = new mapgl.Map(container, {
	center: center,
	maxZoom: maxZoom,
	minZoom: minZoom,
	zoom: zoom,
	maxPitch: maxPitch,
	minPitch: minPitch,
	pitch: pitch,
	rotation: rotation,
	zoomControl: false,
	key: apiKey,
	interactiveCopyright: false,
	autoHideOSMCopyright: autoHideOSMCopyright,
	preserveDrawingBuffer: true
});
setInterval(() => window.map._impl.state.needRerender = true, 300);
```
@webcoderkz you can use this snippet as hot fix, working on better solution.

**Comment by avxkim** [comment]
@teanet where to put this js code?

**Comment by teanet** [comment]
HTML.swift

**Comment by teanet** [comment]
@webcoderkz It should be fixed on our backend side, please check

**Comment by avxkim** [comment]
@teanet okay, i need to install ios14 again, there's no need to update application itself, right? It was fixed just on the backend, no frontend fixes needed?

**Comment by teanet** [comment]
> @teanet okay, i need to install ios14 again, there's no need to update application itself, right? It was fixed just on the backend, no frontend fixes needed?

right

**Comment by avxkim** [comment]
@teanet i can confirm, it works now, thanks!

**Comment by avxkim** [comment]
@teanet i've noticed a bug, map works now, but when i'm reaching certain turn to the left/right, it shows incorrect distance in km, it feels like it was syncing the distance instantly in iOS 13, but now it takes like 5 seconds to re-calculate the correct distance. I hope i explained clearly enough.

**Comment by teanet** [comment]
Ok, I will ask our navigation team

**Comment by Trufi** [comment]
It's pretty hard to understand what's going on and what the problem is without any example, because our SDK doesn't have any about turns and distance calculating. 

@webcoderkz could you provide some example with this problem?
